utils/screenshot_utils.py

```
@@ -0,0 +1,288 @@
+                vision_base_url = core_config.get('VISION_MODEL_URL')
+                vision_model = core_config.get('VISION_MODEL')
+                
+                logger.info(f"自定义API配置 - 密钥配置: {bool(vision_api_key)}, 端点: {vision_base_url}, 模型: {vision_model}")
+                
+                if vision_api_key and vision_base_url and vision_model:
+                    logger.info("使用自定义API视觉模型配置")
+                else:
+                    logger.info("自定义API视觉模型配置不完整，跳过AI分析")
+                    return None
+            else:
+                # 未开启自定义API：使用全局配置的默认设置
```

Ruff (0.14.8)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)